### PR TITLE
RIA-5739 Force case event for Case Building to Case Under Review

### PR DESCRIPTION
**Pull request by gregorio-massara**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by addison-jephtah — DISMISSED** [review]

**Review by addison-jephtah — APPROVED** [review]

**Review by addison-jephtah — APPROVED** [review]

**Comment by addison-jephtah** [comment]
Kindly amend ensure that the integration and unit tests pass before merging

**Review by addison-jephtah — DISMISSED** [review]

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Review by addison-jephtah — DISMISSED** [review]

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
             We don't use the libraries affected by this vulnerability. This is a false positive in dependencycheck that is still current in version 8.2.1.
             Try to remove it when a dependencycheck upgrade becomes available.
-            If it still happens, check that we don't use hutool-json and json-java. If we don't, extend the suppression date by another year.
```
Comment (filippo-possenti-version1) [inline config/owasp/suppressions.xml:19]: Introduced a typo. Probably a mistype?

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeEmailNotificationPersonalisation.java [hunk 2]
```diff
@@ -1,28 +1,41 @@
...
+
+            return Collections.emptySet();
+        }
+
+        if (asylumCase.read(CHANGE_ORGANISATION_REQUEST_FIELD, ChangeOrganisationRequest.class)
+            .map(it -> it.getCaseRoleId() == null)
+            .orElse(false)) {
+
+
+
```
Comment (filippo-possenti-version1) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeEmailNotificationPersonalisation.java:32]: Unnecessary whitespace.

Also: the whole file looks modified instead of specific lines. Did the Line Feed convention change from LF to CRLF? If so, please re-instate LF.

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeEmailNotificationPersonalisation.java [hunk 3]
```diff
@@ -1,28 +1,41 @@
...
+            return Collections.emptySet();
+        } else {
```
Comment (filippo-possenti-version1) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeEmailNotificationPersonalisation.java:34]: The `else` can be omitted here.

The subsequent lines will run anyway, if you don't enter one of the `if` statements. Removing the `else` will help redability.

File: src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/email/AppellantForceCaseProgressionToCaseUnderReviewPersonalisationEmailTest.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.appellant.email;
```
Comment (filippo-possenti-version1) [inline src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/email/AppellantForceCaseProgressionToCaseUnderReviewPersonalisationEmailTest.java:1]: This file is using CRLF as Line Feed convention. Please change it to LF.

Make sure the same applies for all files involved in this PR.
